packages/drawer/src/Drawer.tsx

```diff
@@ -144,15 +149,19 @@ export default function Drawer({
           role="dialog"
           showCloseButton={showCloseButton}
         >
-          <DrawerContent tabIndex={-1}>{children}</DrawerContent>
+          {children}
           {showCloseButton && (
             <DrawerCloseButton
               aria-label={closeButtonTitle}
               data-test="@button:closeDrawer"
               onClick={handleClose}
               title={closeButtonTitle}
             >
-              <CloseIcon aria-hidden fill={shadeBlack} stroke={shadeWhite} />
+              <CloseIcon
+                aria-hidden
```

Comment:
Why it's hidden?

Reply:
Because we want a screen reader to not see the icon but the wrapper button